homeassistant/components/sensor/onewire.py

```diff
@@ -19,21 +30,22 @@
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Setup the one wire Sensors."""
     base_dir = config.get('mount_dir', '/sys/bus/w1/devices/')
```

Comment:
This could be rewritten to `base_dir = config.get(CONF_MOUNT_DIR)` as `voluptuous` is taking care about default.